pkgs/tools/text/difftastic/default.nix

```diff
@@ -2,22 +2,22 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "difftastic";
-  version = "0.6";
+  version = "0.8";
 
   src = fetchFromGitHub {
     owner = "wilfred";
     repo = pname;
     rev = version;
-    sha256 = "WFvxdRCbTBW1RGn2SvAo2iXn82OO/Z06cZQkIu4eiew=";
+    sha256 = "0103py4v4v7xqv85yiczhd9w9h1aa54svhhdibvbl6x4b35y2mk5";
   };
 
-  cargoSha256 = "2hRUfIxNVs4uSrEESas3wvvVsZHVocP8aiO7K0NZ+mY=";
+  cargoSha256 = "1k0d7yadicfzfc2m1aqs4c4a2k3srb54fpwarc3kwn26v3vfjai1";
 
   meta = with lib; {
     description = "A syntax-aware diff";
     homepage = "https://github.com/Wilfred/difftastic";
+    changelog = "https://github.com/Wilfred/difftastic/raw/${version}/CHANGELOG.md";
     license = licenses.mit;
     maintainers = with maintainers; [ ethancedwards8 ];
-    platforms = platforms.unix;
```

Comment:
because `buildRustPackage` already defaults to Rust supported platforms, and reading difftastic code it doesn't seem to restrict its platforms https://github.com/NixOS/nixpkgs/blob/172edcb4268a1cbd0a1eeb683ee4090367aa2c77/pkgs/build-support/rust/default.nix#L152

Reply:
I see.